onnxmltools/convert/coreml/operator_converters/neural_network/Crop.py

```diff
@@ -14,12 +14,22 @@ def convert_crop(scope, operator, container):
     op_type = 'Crop'
```

Comment:
Please drop the check above (https://github.com/onnx/onnxmltools/pull/119/files#diff-eeaf9b91541491801d756404413c846dR11). Thanks.

Reply:
Did not get your point.

Reply:
Oh my bad. Please skip this comment. Thanks.